Source/OxyPlot/Annotations/FunctionAnnotation.cs

```diff
@@ -70,10 +70,10 @@ protected override IList<ScreenPoint> GetScreenPoints()
 
             if (fx != null)
             {
-                double x = this.ActualMinimumX;
+                var x = this.ActualMinimumX;
 
                 // todo: the step size should be adaptive
-                double dx = (this.ActualMaximumX - this.ActualMinimumX) / this.Resolution;
+                var dx = (this.ActualMaximumX - this.ActualMinimumX) / this.Resolution;
```

Comment:
Stuff like this a harrowing: at a glance, I have no idea if this is a real or integer division.

Reply:
Counter-Point: Beginning the line with `double` does not guarantee that a floating-point division is performed. If `ActualMaximumX` and `ActualMinimumX` were `int`s, an integer division would be performed and the result promoted to `double`. I would argue that in this particular case the explicit type could even be misleading.